napps/ofcore/main.py

```diff
@@ -81,7 +81,7 @@ def handle_raw_message_in(self, event):
         self.add_to_msg_in_buffer(new_event)
 
     @ListenTo('KycoMessageInHello')
```

Comment:
The "ListenTo" change was made on another branch that was just merged. So, it is not here for now.